fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java

```diff
@@ -141,7 +142,8 @@ public Cost visitPhysicalFileScan(PhysicalFileScan physicalFileScan, PlanContext
 
     @Override
     public Cost visitPhysicalProject(PhysicalProject<? extends Plan> physicalProject, PlanContext context) {
-        return CostV1.ofCpu(context.getSessionVariable(), 1);
+        long exprCount = physicalProject.getProjects().stream().filter(proj -> !(proj instanceof Slot)).count();
```

Comment:
should check output.size() is same with child's output.size().
otherwise, we may lose some column prune project when select best cost plan